ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/LedgerApiServer.scala

```diff
@@ -36,19 +36,19 @@ trait ApiServer {
 
 }
 
-class LedgerApiServer(
+final class LedgerApiServer(
     createApiServices: (Materializer, ExecutionSequencerFactory) => Future[ApiServices],
     desiredPort: Int,
     maxInboundMessageSize: Int,
     address: Option[String],
-    loggerFactory: NamedLoggerFactory,
     sslContext: Option[SslContext] = None,
     interceptors: List[ServerInterceptor] = List.empty,
     metrics: MetricRegistry,
-)(implicit actorSystem: ActorSystem, materializer: Materializer)
+)(implicit actorSystem: ActorSystem, materializer: Materializer, ctx: LoggingContext)
     extends ResourceOwner[ApiServer] {
+
   override def acquire()(implicit executionContext: ExecutionContext): Resource[ApiServer] = {
-    val logger = loggerFactory.getLogger(this.getClass)
+    val logger = ContextualizedLogger.get[LedgerApiServer]
```

Comment:
Why not make this a field?

Reply:
It's probably a sensible change to make, I'll work on this, thanks.

Reply:
Address https://github.com/digital-asset/daml/pull/4097/commits/55c498bae0abcbf7ad4e81f858347d369aa1d3c6